src/Dictionaries/DictionarySourceHelpers.cpp

```diff
@@ -13,44 +12,54 @@
 
 namespace DB
 {
+
+void formatWithBlock(BlockOutputStreamPtr & out, const Block & block)
```

Comment:
Looks like it can be named just `formatBlock`.

Reply:
Updated.